v1.0_metadata.xml

```diff
@@ -3734,6 +3735,8 @@
         <Property Name="preferredLanguage" Type="Edm.String" />
         <Property Name="proxyAddresses" Type="Collection(Edm.String)" Nullable="false" />
         <Property Name="renewedDateTime" Type="Edm.DateTimeOffset" />
+        <Property Name="resourceBehaviorOptions" Type="Collection(Edm.String)" />
```

Comment:
this file is automatically generated from the metadata downloaded from the service, from the documentation information and from [an XSLT that applies transformations](https://github.com/microsoftgraph/msgraph-metadata/blob/master/transforms/csdl/preprocess_csdl.xsl). It'll be erased weekly.
I can see those properties are missing in the metadata even though they are returned by the service.

You should edit the XLST to insert those nodes automatically to to group entity, and you can test the transformation using the PowerShell script in the same directory.

The transformation will look something like
```xml 
<xsl:template match="edm:Schema[@Namespace='microsoft.graph']/edm:EntityType[@Name='group']">
   <xsl:copy>
       <xsl:copy-of select="@* | node()" />
       <Property Name="resourceBehaviorOptions" Type="Collection(Edm.String)" />
       <Property Name="resourceProvisioningOptions" Type="Collection(Edm.String)" />
       <Property Name="creationOptions" Type="Collection(Edm.String)" />
   </xsl:copy>
</xsl:template>
```